### ISSUE TITLE
Implement web page for spring re-engage campaign FE

### ISSUE THREAD

**Issue by meeragovindaraju** [openstax/bit#203]
# Description
The spring re-engage campaign is the most important campaign that the marketing team runs every year. This year, the challenge is presenting the OpenStax journey to potential adopters in a fresh and unique way. We want to make our internal adoption milestones meaningful, interesting, and fun for our prospective adopters. 

Using the tools available to us, we can personalize the OpenStax experience, guiding them all the way from account creation to adoption.

Using Accounts, Salesforce, Pardot, and OpenStax.org, we will deliver a personalized experience to contacts in the campaign. By logging into our accounts system from a link delivered through email, contacts will be able to see where they are on the OpenStax adoption journey and be encouraged to finish the journey.

# Acceptance
Non-adopting OpenStax account holders and highly engaged non-account holders receive an email linking them to openstax.org/hero-journey. Users on openstax.org cannot access the page without the email link.

Users are prompted to log in (if they're not already logged in) or create an account to see where they are in the OpenStax journey.

Users are asked to look at the books. They can either go look at the books from a link on the page, or click a link that indicate they have already looked at the book. Once they click that second link, they can move on to the next stage of the page.

In the next stage, users are presented with a pop quiz. They must answer all three questions to move on to the next stage of the journey. Once they've answered the questions, they are able taken to the adoption part of the journey. **If users do not want to take the pop quiz, there is a fast track to the adoption part of the journey. If they click on this link, they are taken to the adoption section of the page (not the adoption form).**

At the adoption section, users should be able to click on the adoption form and adopt in that same session. If they adopted in a previous session, this should be reflected on the page.

Once the adoption has been submitted, there should be no downtime to move on to the next and final stage of the page. In the final stage, users should be able to submit a testimonial. Once they hit submit, the badge should be unlocked a message should appear telling them that they received their badge in their email, and giving them the option to give us their mailing address and have it mailed to them. 

# Checklist for done
2B-UX DESIGN
- [x] Develop hi-fi mockup with updated UI/UX elements
- [x] Get UX team’s input on design
- [x] Review initial design with Allison
- [x] Review design with dev team and adjust as needed
- [x] Get final design approved by Allison

**Comment by Malar-Natarajan** [comment]
@RoyEJohnson @allisonhowell Documenting the issues that I found in this card.
SHA used: 73b44ab12effd3ee5c1ad9ea81ed53225b5837ab

1.  For flows like, new faculty sign-up, existing faculty login, existing non-faculty sign-up - I am kind of blocked after step 4. I submit adoption form and when I come back to hero page the journey bar takes me back to step 2
Only if I signed-up as a new "student" and then attempted quiz, completed adoption form - I am able to pass through the testimonial page and finally the hero page. 

2. in safari, every time a step is completed the following section scrolls to the bottom automatically. I have to manually scroll up to see the beginning of the section

3. In safari, we see trailing images (ghost images?) when we click on each of the journey bar points 

4. Without redirects from Subject and atleast adoption page, I dont think users would come back to the Hero page to give a testimonial.

5. Not sure if we will be sending these campaign emails multiple times to the same users. When I re-login with a user who had already got the hero badge, the page lands me again in the testimonial page instead of the final hero page. Not sure if that was intentional
